docs/source/use_cases/run_picard_and_gatk/index.rst

```diff
@@ -17,20 +17,178 @@ __ RenderedVersion_
 
 .. comment: end: goto-read-the-docs
 
-`Picard`_ and `GATK`_ tools are popular utilities used for genomics analysis and
-processing pipelines.
+Introduction
+============
+`Picard <http://broadinstitute.github.io/picard/>`_/`GATK <https://www.broadinstitute.org/gatk/>`_ tools are command line utilities for genomic sequencing data processing that typically take BAM and other files as input and produce modified BAM files.
 
-They usually work with BAM files but we are working on teaching them to work
-with the genomic data in the cloud.
+These tools are frequently chained together into pipelines to perform step-by-step processing of the sequencing data all the way from unaligned sequencer output to variant calls (e.g. see `Broad best practices <https://www.broadinstitute.org/gatk/guide/best-practices>`_). 
 
-If your dataset is loaded into a cloud provider supporting `GA4GH`_ API
-(e.g. `Google Genomics`_) or you want to use one of
-the available datasets from :doc:`/use_cases/discover_public_data/index`,
-you can check out `gatk-tools-java`_
-library that makes it possible to run some Picard tools against genomic data
-in the cloud.
+=====
 
-It is now also possible to make a special build of `Picard`_ tools
-that will have this support built in by default.
+We are teaching these tools to take cloud based datasets as a possible input. The foundation for cloud data access is now in HTSJDK library and we have converted a number of Picard tools.
+
+If your dataset is loaded into a cloud provider supporting `GA4GH <http://ga4gh.org/#/>`_ API (e.g. `Google Genomics <https://cloud.google.com/genomics/>`_) or you use one of the available datasets from `Discover Published Data <http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_public_data/index.html>`_, you will be able to run a Picard tool against it, reading data directly from the cloud.
```

Comment:
Links!

For within RTD links, you can use :doc:`/use_cases/discover_public_data`

For commonly used external links, we keep them here https://github.com/googlegenomics/readthedocs/blob/master/docs/source/conf.py#L127 so that when we need to update a link, it only happens in one place.